SharedCode/src/commonMain/kotlin/ApplicationPresenter.kt

```diff
@@ -88,7 +94,15 @@ class ApplicationPresenter : ApplicationContract.Presenter() {
 
     private fun convertToPriceString(priceInPennies: Int?): String {
         if (priceInPennies == null) return "sold out"
-        return "from £${priceInPennies / 100}.${priceInPennies % 100}"
+        return "from £${priceInPennies / 100}.${padEnd("${priceInPennies % 100}", "0", 2)}"
```

Comment:
I think this reads clearly, but you could perhaps change this to something like:
```
return priceInPennies?.let {
    "from £${it / 100}.${padEnd("${it % 100}", "0", 2)}"
} ?: "sold out"
```